packages/chart-advisor/src/utils.ts

```diff
@@ -5,3 +5,9 @@ export function hasSubset(array1: any[], array2: any[]): boolean {
 export function intersects(array1: any[], array2: any[]): boolean {
   return array2.some((e) => array1.includes(e));
 }
+
+export function deleteProperty(source: Record<string, any>, property: string) {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { [property]: _, ...rest } = source;
+  return rest;
+}
```

Comment:
what's the diff with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete

Reply:
@pddpd said to use destructing instead of delete as far as possible in the previous resolved comments, I thought delete was harmful to the code... no?

Reply:
In fact, we could use `delete` or any other delete method with personal preference. But some opinions believed `delete` may make confusion.

Using destructuring instead of delete is a suggestion. It's just like using const instead var. It's not a required option. I'm a little worried about the confusion caused by `delete`. For example, 
```
import constA from 'const';
let varB = constA;
delete constA['a'];
```
Then, all the `constA` will lost property `a`. (Of course, we don't have this problem here, because CKBJson is a function.)

If we know exactly what we're doing, I will support using `delete`.

What's ur opinion? @neoddish @soundquiet

Reply:
in this case, I would like to use `delete` for `destructuring` creating new variables.